dependencies.props

```diff
@@ -37,8 +37,8 @@
     <ProjectNTfsExpectedPrerelease>beta-26007-00</ProjectNTfsExpectedPrerelease>
     <ProjectNTfsTestILCExpectedPrerelease>beta-26007-00</ProjectNTfsTestILCExpectedPrerelease>
     <ProjectNTfsTestILCPackageVersion>1.0.0-beta-26007-00</ProjectNTfsTestILCPackageVersion>
-    <MicrosoftNETCoreDotNetHostPackageVersion>2.0.1-servicing-25615-03</MicrosoftNETCoreDotNetHostPackageVersion>
-    <MicrosoftNETCoreDotNetHostPolicyPackageVersion>2.0.1-servicing-25615-03</MicrosoftNETCoreDotNetHostPolicyPackageVersion>
+    <MicrosoftNETCoreDotNetHostPackageVersion>2.1.0-preview1-25929-07</MicrosoftNETCoreDotNetHostPackageVersion>
```

Comment:
When https://github.com/dotnet/core-setup/issues/3510 is fixed, can we add these new properties to the steps below so @dotnet-maestro-bot will update them automatically?

Reply:
Yes, that would be great. Let me create an issue for that so that we don't loose track of this.

Reply:
Issue created - #25839